micro_ros_diagnostic_bridge/src/micro_ros_diagnostic_bridge/micro_ros_diagnostic_bridge.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2021 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/microros/system_modes
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "micro_ros_diagnostic_bridge/micro_ros_diagnostic_bridge.hpp"
+
+#include <string>
+#include <memory>
+#include <utility>
+
+#include <rcl/error_handling.h>
+
+#include <rclcpp/parameter.hpp>
+#include <rclcpp/parameter_map.hpp>
+#include <rcl_yaml_param_parser/parser.h>
+
+using micro_ros_diagnostic_msgs::msg::MicroROSDiagnosticStatus;
+using diagnostic_msgs::msg::DiagnosticStatus;
+
+MicroROSDiagnosticBridge::MicroROSDiagnosticBridge(const std::string & path)
+: Node("micro_ros_diagnostic_bridge"),
+  logger_(rclcpp::get_logger("MicroROSDiagnosticBridge"))
+{
+  // Read lookup table
+  declare_parameter("lookup_table", rclcpp::ParameterValue(path));
+  std::string lookup_table_path = get_parameter("lookup_table").as_string();
+  if (lookup_table_path.empty()) {
+    throw std::invalid_argument("Need path to lookup table.");
+  }
+  read_lookup_table(lookup_table_path);
+
+  auto callback =
+    [this](const MicroROSDiagnosticStatus::SharedPtr msg_in) -> void
+    {
+      RCLCPP_DEBUG(
+        get_logger(),
```

Comment:
```suggestion
        logger_,
```
Or remove `logger_` at all.